src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+
+    }
```

Comment:
Please ensure that the main method contains instantiation and method calls for the different `Machine` objects as per the task requirements. For example, create instances of `Truck`, `Bulldozer`, and `Excavator`, and call their respective methods to demonstrate polymorphism.